plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/condition/ShellMatchingMatcherIsAvailable.java

```diff
@@ -39,13 +38,20 @@ public ShellMatchingMatcherIsAvailable(Matcher<?>... matchers) {
 		AndMatcher am  = new AndMatcher(matchers);
 		this.matcher = am;
 	}
+	
+	/**
+	 * Creates new ShellMatchingMatcherIsAvailable wait condition with specified shell text.
+	 * @param text shell text
+	 */
+	public ShellMatchingMatcherIsAvailable(String text){
```

Comment:
We already have ShellIsAvailable.

ShellIsAvailable is on swt.layer and takes RedDeer Shell
ShellMatchingMatcherIsAvailable is on core.layer and accepts String or Matcher. It cannot accept RedDeer Shell since core layer is too low.

So either we keep the matchers as they are now and find more sutiable name for ShellMatchingMatcherIsAvailable or we merge them to one - ShellIsAvailable, move them to core layer and the matcher will no longer accept RedDeer Shell but SWT Shell. WDYT ?